config/initializers/exception_notification.rb

```diff
@@ -0,0 +1,14 @@
+require 'exception_notification/rails'
+require 'exception_notification/sidekiq'
+
+EXCEPTION_RECIPIENTS = JSON.parse(Figaro.env.exception_recipients).freeze
+
+ExceptionNotification.configure do |config|
+  config.add_notifier(
+    :email,
+    email_prefix: "[#{APP_NAME} EXCEPTION - #{Figaro.env.domain_name}] ",
```

Comment:
Still doesn't have the environment? We won't be able to tell staging apart from production?

We can use `LoginGov::Hostdata.env` for that

Reply:
`domain_name` is unique per environment. In production, it is `secure.login.gov`. In staging, it is `idp.staging.login.gov`.

Reply:
Oh, by `env`, you mean the short name, like `staging`, `dev`, `int`?

Reply:
Fixed here: https://github.com/18F/identity-idp/pull/1740/commits/7a67adb33dd92a924ed5d6f68a3156141c2b4566